stellarphot/gui_tools/comparison_functions.py

```diff
@@ -560,10 +505,12 @@ def _viewer(self):
 
         legend = ipw.HTML(
             value="""
-        <h3>Green circles -- Gaia stars within 2.5 arcmin of target</h3>
-        <h3>Red circles -- APASS stars within 1 mag of target</h3>
-        <h3>Blue circles -- VSX variables</h3>
-        <h3>Red × -- Exclude as target or comp</h3>
+        <ul>
+        <li>Green circles -- Gaia stars within 2.5 arcmin of target</li>
+        <li>Red circles -- APASS stars within 1 mag of target</li>
```

Comment:
OK, I'm going to be difficult here, since I should have thought of this earlier.  A not insignificant subset of the population is color-blind... can't we use different shapes as well as colors?

Reply:
I'm getting to open an issue for this -- I think the markers are also maybe too small. You are correct though that straight up using red/green is about the worst possible choice from an accessibility POV.